include/cli/modules/deploy.php

```diff
@@ -277,8 +276,12 @@ function run($args, $options) {
                 "*/.htaccess", ".MANIFEST"));
         }
 
-        if (!$options['dry-run'])
-            $this->writeManifest($this->destination);
+        if (!$options['dry-run']) {
+            $this->writeManifest();
+
+            $this->clean($root, $this->destination, -1, array($include, "setup/"));
+            $this->clean("$root/include", $include, -1, array("ost-config.php","settings.php","plugins/", "*/.htaccess"));
```

Comment:
Hmm, I only added the dry run, maybe something went wrong with merging, I will take a look at this!